### PR TITLE
Analysis (DE and EN): Text adjustments

### PR DESCRIPTION
**Pull request by HelTob**
Some text adjustments for clarification.

### CODE REVIEW

File: src/data/analyse_en.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
...
...ser takes more time to share his or her test result. As soon as the result is shared, the number of shared results changes accordingly regarding the scan or call date.<br />The user already receives a QR code when making their appointment and scans it immediately, although he or she may not have the test performed until later.</p><strong>Additional information</strong><p>Only positive test results are relevant for ending chains of infection. To prevent abuse, positive test results must be verified. There are two options for verifying test results: a QR code or a teleTAN, which must be entered in the Corona-Warn-App. The user receives the QR code when the sample for the test is collected. By scanning the QR code, the test can be registered in the Corona-Warn-App. As soon as the result is available, it is automatically retrieved and displayed on the smartphone. The user can then decide to release her/his own random codes from the last 14 days to warn possible risk contacts. If no QR code is available or if it is lost, the user can call the verification hotline. A teleTAN is generated there to verify the positive test result. This teleTAN must be entered into the app by the user.</p>"
```
Comment (larswmh) [inline src/data/analyse_en.json:123]: destribution -> distribution

**Review by larswmh — CHANGES_REQUESTED** [review]
One comment added about a typographical error found in the English text, the rest looks good to me

File: src/data/analyse_en.json (hunk 1 above)
Comment (HelTob) [reply src/data/analyse_en.json]: Thanks!

**Review by larswmh — APPROVED** [review]
LGTM

**Comment by MikeMcC399** [comment]
@HelTob 

I also noticed the typo that @larswmh mentioned so I used the Spell Right spell checker in VS Code and there is another typo in the file
https://github.com/corona-warn-app/cwa-website/blob/6fff7a3720a068dc73b408b116512337799cf1e1/src/data/analyse_en.json#L7

"calender" should be "calendar" in English. Maybe you could correct this line as well?

----

For future PRs it is better for git handling if you create a new branch in your fork for changes and not make the changes directly in your `master` branch. See https://github.com/corona-warn-app/cwa-website/blob/master/CONTRIBUTING.md#pull-request-checklist.

The checklist also recommends squashing commits if you make corrections to the same file. If you do that you do need to force push to GitHub afterwards.

**Comment by HelTob** [comment]
@MikeMcC399 Thank you very much for your comment. 

I tried to create a new branch in my fork and add the changes but it seems like I made something wrong and accidentally created a new PR (#2964). 

Best regards 
Tobias

**Comment by MikeMcC399** [comment]
@HelTob 
- You could close this PR #2963 and use only the new PR #2964 since it now covers all your changes.
- If you do that, you would probably want to rename #2964.

**Comment by HelTob** [comment]
@MikeMcC399 

Thanks again, Mike. I will educate myself about not making changes directly in my master and squashing commits.

Best regards,
Tobias

**Pull request closed without merging**